core/templates/dev/head/pages/base.html

```diff
@@ -119,6 +119,10 @@
       {% endif %}
     </script>
 
+    <!-- Note to developers: This script needs to be loaded before external
```

Comment:
I thought it would be more organized, but yes in terms of efficiency it would be better inline here.